test/index.html

```diff
@@ -17,19 +17,19 @@
 
     <script>
       WCT.loadSuites([
-        'basic.html',
-        'horizontal.html',
-        'a11y.html',
-        'nested.html',
-        'flex.html',
-        'basic.html?dom=shadow',
-        'horizontal.html?dom=shadow',
-        'a11y.html?dom=shadow',
-        'nested.html?dom=shadow',
-        'flex.html?dom=shadow'
+        'basic.html?wc-shadydom=true&wc-ce=true', //shady
+        'horizontal.html?wc-shadydom=true&wc-ce=true', //shady
+        'a11y.html?wc-shadydom=true&wc-ce=true', //shady
+        'nested.html?wc-shadydom=true&wc-ce=true', //shady
+        'flex.html?wc-shadydom=true&wc-ce=true', //shady
+        'basic.html?dom=shadow', //shadow
```

Comment:
don't think this `//shadow` comment is needed, since `dom=shadow` is pretty self explaining :)